schema/commands.json

```diff
@@ -44,6 +41,19 @@
   "title": "jupyterlab-jhub-apps",
   "description": "jupyterlab-jhub-apps custom command settings.",
   "type": "object",
-  "properties": {},
+  "properties": {
```

Comment:
I moved these to `properties` as it was the only way that I was able to modify them at runtime (this was useful for testing different values). See https://github.com/jupyterlab/extension-examples/tree/main/settings.

I also realised that if a user updated their settings (e.g. changed the `baseUrl`), then this would not be reflected in the application. I have using properties and a change signal (`settings.changed.connect` below) to fix this.